tests/network/connectivity/test_ovs_linux_bridge.py

```diff
@@ -11,6 +11,7 @@
 
 class TestConnectivityLinuxBridge:
     @pytest.mark.gating
+    @pytest.mark.conformance
```

Comment:
This scenario requires the nmstate operator to be installed on the cluster. Are we OK with this for conformance testings?

Reply:
already removed, missed to push